client/blocks/app-banner/index.jsx

```diff
@@ -201,6 +214,7 @@ const mapStateToProps = state => {
 	return {
 		dismissedUntil: getPreference( state, APP_BANNER_DISMISS_TIMES_PREFERENCE ),
 		currentSection: getCurrentSection( sectionName, isNotesOpen ),
+		currentPathFragment: getCurrentPathFragment( state ),
```

Comment:
Consider passing through  `getCurrentRoute( state )` instead, and letting the getDeepLink function(s) strip out the leading slash. It's much easier to read with that context, instead of needing to click through and see what a current path fragment is.

```
const route = getCurrentRoute( state ); //this may be null

Reply:
Thanks, I've removed this dependency and added some null checking.